app/views/devise/registrations/new.html.erb

```diff
@@ -12,6 +12,11 @@
           <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
         </div>
 
+        <div class="field" >
+          <%= f.label "Full name" %><br />
+          <%= f.text_field :name, placeholder: "J. Doe", autocomplete: "name" %>
```

Comment:
Wrong order (for materialize), and we need a name symbol in there.  Also, the edit page needs a field like this too.